### ISSUE TITLE
BUG: `()` must be used when there're more than 1 statement in `if`

### ISSUE THREAD

**Issue by iredmail** [NikolaLohinski/gonja#19]
- NOT work: `{% if ... or ... %}` (No `()` before `or`)
- Works: `{% if (...) or ... %}`  (Has `()` before `or`)
- Works: `{% if (...) or (...) %}`  (Has `()` before `or`)

```
package main

import (
	"fmt"
	"os"

	"github.com/nikolalohinski/gonja/v2"
	"github.com/nikolalohinski/gonja/v2/exec"
)

var (
	tmpl1 = `{% if var1 in ['1', '2'] or (var2 == '3') %}ok{% endif %}`
	tmpl2 = `{% if (var1 in ['1', '2']) or var2 == '3' %}ok{% endif %}`
	tmpl3 = `{% if (var1 in ['1', '2']) or (var2 == '3') %}ok{% endif %}`
)

func main() {
	m := exec.NewContext(map[string]interface{}{
		"var1": "1",
		"var2": "3",
	})

	t1, _ := gonja.FromString(tmpl1)
	err1 := t1.Execute(os.Stdout, m)
	fmt.Printf("error: %v\n", err1)

	t2, _ := gonja.FromString(tmpl2)
	err2 := t2.Execute(os.Stdout, m)
	fmt.Printf("\nerror: %v\n", err2)

	t3, _ := gonja.FromString(tmpl3)
	err3 := t3.Execute(os.Stdout, m)
	fmt.Printf("\nerror: %v\n", err3)
}
```

Command output:
```
default
ERRO[0000] Value.Contains() not available for type: bool 
error: <nil>
ok
error: <nil>
ok
error: <nil>
```

**Comment by NikolaLohinski** [comment]
@iredmail fixed and released in `v2.2.3` 

Thanks a ton for the report :bow:

**Comment by iredmail** [comment]
@NikolaLohinski Thanks for the great work. :)
